pkg/await/await.go

```diff
@@ -229,51 +253,49 @@ func Update(
 	// - [ ] Support server-side apply, when it comes out.
 	//
 
-	// Retrieve live version of last submitted version of object.
-	clientForResource, err := client.FromResource(pool, disco, lastSubmitted)
+	client, err := c.ClientSet.ResourceClient(c.Previous.GroupVersionKind(), c.Namespace)
 	if err != nil {
 		return nil, err
 	}
 
 	// Get the "live" version of the last submitted object. This is necessary because the server may
 	// have populated some fields automatically, updated status fields, and so on.
-	liveOldObj, err := clientForResource.Get(lastSubmitted.GetName(), metav1.GetOptions{})
+	liveOldObj, err := client.Get(c.Previous.GetName(), metav1.GetOptions{})
 	if err != nil {
 		return nil, err
 	}
 
 	// Create merge patch (prefer strategic merge patch, fall back to JSON merge patch).
 	patch, patchType, err := openapi.PatchForResourceUpdate(
-		disco, lastSubmitted, currentSubmitted, liveOldObj)
+		c.ClientSet.DiscoveryClientCached, c.Previous, c.Inputs, liveOldObj)
 	if err != nil {
 		return nil, err
 	}
 
-	// Issue patch request. NOTE: We can use the same client because if the `kind` changes, this
-	// will cause a replace (i.e., destroy and create).
-	currentOutputs, err := clientForResource.Patch(currentSubmitted.GetName(), patchType, patch)
+	// Issue patch request.
+	// NOTE: We can use the same client because if the `kind` changes, this will cause
+	//		 a replace (i.e., destroy and create).
```

Comment:
Weird tab here -- I've seen this before in your PRs so my guess is that your editor (JetBrains?) does "stuff", lol. Would you mind double checking that there aren't other issues like this?

Reply:
Can you use `gofmt` in your editor? With the general styling & format comments in this PR, I think we should setup go formatting, linting, etc in a git commit hook. Thoughts?

Reply:
I ran this through `goimports` already which should also run `gofmt`. I can't remember if the build toolchain includes that, but I agree that we should set it up if not.